src/engine/strat_engine/backstore/device.rs

```diff
@@ -127,21 +127,19 @@ pub fn is_stratis_device(devnode: &Path) -> StratisResult<Option<PoolUuid>> {
 }
 
 #[cfg(test)]
-// rustfmt bug requires this?
-#[cfg_attr(rustfmt, rustfmt_skip)]
 mod test {
     use std::path::Path;
 
-    use super::super::super::cmd::{create_ext3_fs, udev_settle};
+    use super::super::super::cmd;
```

Comment:
These changes related to adding cmd::xxx are intentional?  They caused problems with rustfmt?

Reply:
rustfmt, for some reason, wasn't alphabetizing ```create_ext3_fs, udev_settle``` in the proper order. It was getting them backwards. This avoids having the imports that puzzle it so, rather than turning off rustfmt for the entire module.

Reply:
Ok.  Thanks. :+1: